tests/hash-backend-integration/src/tests/graph/knowledge/primitive/entity.test.ts

```diff
@@ -149,6 +152,20 @@ describe("Entity CRU", () => {
     });
   });
 
+  afterAll(async () => {
+    await deleteKratosIdentity({
+      kratosIdentityId: testUser.kratosIdentityId,
+    });
+    await deleteKratosIdentity({
+      kratosIdentityId: testUser2.kratosIdentityId,
+    });
```

Comment:
Resetting the test users is not strictly needed but it's cleaner.

Reply:
Probably not necessary for us to do right now, but I assume manually keeping track of created/deleted users may be somewhat annoying/error prone. Perhaps we should have `createTestUser` persist a list of created users per-test so we can just loop through the `kratosIdentityId`s in these `afterAll`s

Reply:
Yeah, 100% agreed!